apps/minaintyg/src/schema/info.schema.ts

```diff
@@ -0,0 +1,20 @@
+import { z } from 'zod'
+
+export enum BannerPriority {
+  ERROR = 'ERROR',
+  OBSERVE = 'OBSERVE',
+  INFO = 'INFO',
+}
+
+export const bannerTypeEnum = z.nativeEnum(BannerPriority)
+
+export const bannerSchema = z.object({
+  content: z.string(),
+  type: bannerTypeEnum,
+})
+export const infoSchema = z.object({
+  banners: z.array(bannerSchema),
+})
+
+export type Info = z.infer<typeof infoSchema>
```

Comment:
Should info be named something less general like InformationResponse or similar?

Reply:
Good idea!